kubernetes/pods/spinnaker-helm/ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Is there a better set of permissions to give spinnaker rather than just the `cluster-admin` role?

Reply:
The deployment does not specify any  service account, so it basically uses the default of the namespace: https://github.com/helm/charts/blob/master/stable/spinnaker/templates/deployments/deck.yaml

Reply:
Yeah...the helm chart might not have it but Spinnaker itself talks about it:  https://www.spinnaker.io/setup/install/providers/kubernetes-v2/#optional-configure-kubernetes-roles-rbac